src/frontend/main/map/DorlingCartogram.tsx

```diff
@@ -0,0 +1,332 @@
+import React, { useEffect, useState } from 'react';
+import * as d3 from 'd3';
+import * as topojson from 'topojson';
+import Modal from '../Modal';
+import ModalContent from '../ModalContent';
+import useModal from '../useModal';
+
+let mapNode!: SVGSVGElement | null;
+
+interface mapProperties {
```

Comment:
This should probably be imported from the other file in which it was defined, instead of being re-defined here..?